### PR TITLE
feat: `test` directory - readme

### PR DESCRIPTION
**Pull request by tkonieczny**

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+│   ├── cypress-13
+│   └── cypress-14
+...
+├── junit-pregenerated-reports # additional custom cases for junit reports (mainly edge-cases)
+│   ├── crd-workflow
+│   ├── high-level-failure.xml
+│   ├── high-level-testcase-both-error-and-failure.xml
+│   └── high-level-without-testcases.xml
+...
+├── legacy # Executors, Tests, TestSuites and other legacy things
```
Comment (olensmar) [inline test/README.md]: can we add a link here to https://docs.testkube.io/articles/legacy-features ?

Comment (tkonieczny) [reply test/README.md]: added
